### PR TITLE
Erik

### PR DESCRIPTION
**Pull request by OneZeroWasTaken**

### CODE REVIEW

File: homeassistant/components/plex/__init__.py [hunk 1]
```diff
@@ -117,6 +117,55 @@ def gdm_scan():
...
+                _LOGGER.error(
+                    "Websocket to %s failed, aborting [Error: %s]",
+                    entry.data[CONF_SERVER],
+                    error,
+                )
+                hass.async_create_task(hass.config_entries.async_reload(entry.entry_id))
+
+        elif msgtype == "playing":
+            hass.async_create_task(plex_server.async_update_session(data))
+        elif msgtype == "status":
```
Comment (Sebulon) [inline homeassistant/components/plex/__init__.py]: Should lower the cognitive complexity by inverting the if statements, I calculated to 19 in cognitive complexity

File: homeassistant/components/plex/__init__.py [hunk 2]
```diff
@@ -117,6 +117,55 @@ def gdm_scan():
     return True
 
 
+def _get_plex_account(plex_server):
+    try:
+        return plex_server.account
+    except (plexapi.exceptions.BadRequest, plexapi.exceptions.Unauthorized):
+        return None
+
+def setup_websocket(hass, entry, plex_server, server_config, server_id):
```
Comment (Sebulon) [inline homeassistant/components/plex/__init__.py]: No underscore... Not following convention

**Review by Sebulon — CHANGES_REQUESTED** [review]
Fix the two comments, and then this is good for merging

File: homeassistant/components/plex/__init__.py (hunk 1 above)
Comment (OneZeroWasTaken) [reply homeassistant/components/plex/__init__.py]: Done

File: homeassistant/components/plex/__init__.py (hunk 2 above)
Comment (OneZeroWasTaken) [reply homeassistant/components/plex/__init__.py]: Done

**Review by OneZeroWasTaken — COMMENTED** [review]
I don't understand how pr's work. Do I have to review my changes in order to show that I resolved the comments? ¯\_(ツ)_/¯

**Comment by OneZeroWasTaken** [comment]
How do I remove "Changes requested" please remove it someone who knows thanks

**Review by Sebulon — APPROVED** [review]

**Pull request merged**